pkg/build/api/register.go

```diff
@@ -1,14 +1,17 @@
 package api
 
 import (
+	kapi "k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/runtime"
+	"k8s.io/kubernetes/pkg/watch/versioned"
 )
 
-const GroupName = ""
-const FutureGroupName = "build.openshift.io"
+const LegacyGroupName = ""
+const GroupName = "build.openshift.io"
```

Comment:
This is likely to have interesting effects on bootstrap policy.  I think we actually get away with for reconciliation purposes since new clusters won't care and old clusters will be additive only.  @liggitt that's neat, isn't it?